cfpb/design-manual
# Update for capital-framework 5.2.0

Updates the project to use the latest release from Capital Framework.

## Changes

- Updates cf-forms to 5.1.0
- Updates cf-icons to 4.4.1

## Testing

- Run `npm start`
- Open `http://localhost:4000/page-components/form-fields.html#notifications` and view the field level error (You'll have to edit the html in the inspector to make it break to two lines).
- Open `http://localhost:4000/page-components/form-fields.html#dropdowns` and inspect the height of the select elem, it should be 35px.

## Screenshots

<img width="366" alt="screen shot 2018-07-11 at 9 37 15 am" src="https://user-images.githubusercontent.com/1280430/42579466-019a7f96-84ee-11e8-8253-074ee2cab582.png">

<img width="399" alt="screen shot 2018-07-11 at 9 38 53 am" src="https://user-images.githubusercontent.com/1280430/42579560-3a8ec92e-84ee-11e8-8f80-3258554388b0.png">
